contrib/depends/packages/packages.mk

```diff
@@ -1,6 +1,14 @@
+ifeq ($(host_os),android)
+packages:=boost openssl zeromq cppzmq libiconv
```

Comment:
cppzmq is now not needed anymore

Reply:
Ok. Then it should be removed for all platforms, right?

Reply:
It is still referenced multiple times in README.md by the way

Reply:
AFAIK no platform needs it anymore.